tools/local/src/db.ts

```diff
@@ -23,16 +23,25 @@ export async function prepareDatabase(url?: string): Promise<{
     const cwd = path.join(__dirname, "../../../internal/db");
 
     await new Promise((resolve, reject) => {
+      let stdoutData = "";
       const p = exec("pnpm drizzle-kit push", {
         env: {
           DRIZZLE_DATABASE_URL: url ?? "mysql://unkey:password@localhost:3306/unkey",
           ...process.env,
         },
-
         cwd,
       });
+
+      p.stdout?.on("data", (data) => {
+        stdoutData += data;
+      });
+
       p.on("exit", (code) => {
         if (code === 0) {
+          if (stdoutData.includes("Error")) {
```

Comment:
stderr is empty and the exit code does not show failure. so I am scanning stdout for a migration failed message